src/valid/expression.rs

```diff
@@ -1518,4 +1422,32 @@ impl super::Validator {
         };
         Ok(stages)
     }
+
+    fn global_var_ty(
+        module: &crate::Module,
+        function: &crate::Function,
+        ty: Handle<crate::Expression>,
```

Comment:
I think this argument should be named `expr`, not `ty`.

Reply:
Ah, good catch. Not sure why I did that. 😓 Fixed with c692bd6a.